infra/bridge/src/controllers/promotions.js

```diff
@@ -26,13 +29,21 @@ const waitFor = timeInMs =>
   new Promise(resolve => setTimeout(resolve, timeInMs))
 
 router.post('/verify', verifyPromotions, async (req, res) => {
-  const { type, socialNetwork, identity, content } = req.body
+  const { type, socialNetwork, identity, identityProxy, content } = req.body
 
   const attestation = await Attestation.findOne({
     where: {
-      ethAddress: identity.toLowerCase(),
+      [Sequelize.Op.or]: [
+        {
+          ethAddress: identity.toLowerCase()
+        },
+        {
+          ethAddress: identityProxy.toLowerCase()
```

Comment:
In some case there may not be a proxy ?
Update the query to handle that case. Maybe a IN clause could make it simpler. Something like this:
```
const addresses = [ identity.toLowerCase() ]
if (identityProxy) addresses.push(identityProxy.toLowerCase()
Attestation.findOne({ ethAddress: { [Sequelize.Op.in]: addresses } }, ...)
```